variables.tf

```diff
@@ -12,6 +12,12 @@ variable "domain" {
   description = "Route53 Domain to manage DNS under"
 }
 
+variable "private_zone" {
+  type        = "string"
+  description = "this is used to if route53 zone is private or not"
```

Comment:
Ditto with the other description, could you change it?

Reply:
here too